.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```diff
@@ -0,0 +1,86 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to file a bug report!
+
+        Before posting, please check that the bug hasn't already been:
+        * [fixed in the next release](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
+        * [discussed previously](https://github.com/stylelint/stylelint/search)
+
+        You can help us fix the bug more quickly by:
+        1. Figuring out what needs doing and proposing it in the issue.
+        2. After the bug is confirmed, [contributing](https://github.com/stylelint/stylelint/blob/main/CONTRIBUTING.md) a pull request.
+  - type: textarea
+    id: reproduce-bug
+    attributes:
+      label: 'What steps are needed to reproduce the bug?'
+    validations:
+      required: true
+  - type: textarea
+    id: stylelint-configuration
+    attributes:
+      label: 'What Stylelint configuration is needed to reproduce the bug?'
+      description: 'Provide the smallest reproducible example.'
+      placeholder: |
+        e.g.
+
```

Comment:
@jeddy3 Is this `e.g.` also no longer needed?

```suggestion
```

Reply:
Yeap, that goes too.